ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceImpl.scala

```diff
@@ -63,7 +63,7 @@ private[index] class IndexServiceImpl(
     transactionsReader: LedgerDaoTransactionsReader,
```

Comment:
Just food for thought for the future: I am not entirely sold on the concept that we allow a stale API to serve the requests, we might want to investigate more what it would mean for individual endpoints to act upon stale data.